src/Contentstack.php

```diff
@@ -30,6 +30,7 @@
  * @license   https://github.com/contentstack/contentstack-php/blob/master/LICENSE.txt MIT Licence
  * @link      https://pear.php.net/package/contentstack
  */
+#[\AllowDynamicProperties]
```

Comment:
Is `#[\AllowDynamicProperties]` a deliberate part of the code or something that was written unintentionally?

Reply:
#[AllowDynamicProperties] is a new attribute added in PHP 8.2, that avoids the deprecation notice on dynamic properties.